src/AzurePipelinesToGitHubActionsConverter.Core/GitHubActions/Step.cs

```diff
@@ -19,11 +17,14 @@ public string run
             }
             set
             {
-                //Spaces on the beginning or end seem to be a problem for the YAML serialization
                 if (string.IsNullOrEmpty(value) == false)
                 {
-                    value = value.Trim();
+                    // Spaces on the beginning or end seem to be a problem for the YAML serialization, so we Trim() here
+                    //  Also, accidental carriage returns in scripts (such as a path including a \r) need to be accounted for
```

Comment:
@NateRickard nit - still 2 spaces here 😿

Reply:
First of all, dang you're setting the bar high for PR acceptance :sweat_smile:

This was actually intentional, I kinda like this for multiline comments - it reads like a paragraph. Does this offend that much considering the majority of the older comments have no spacing at all?